### PR TITLE
Added letter based avatar in case of blank display picture

### LINKED ISSUE #1345: Letter based avatar in case of blank display picture
**Is your feature request related to a problem? Please describe.**
Currently, any organization without any display image have blank squares on My Organizations page while it is letter based everywhere in the website which includes dashboard page and user profile whenever no image is selected.

**Describe the solution you'd like**
Show letter based avatar for organizations based on their initials without any image similar to what we are shown on the dashboard page.

**Approach to be followed (optional)**
We will use dicebear api, similar to how it is being used in LeftDrawerEvent component in dashboard as of now.

**Additional context**
Current condition:
talawa

While the dashboard shows letter based avatar.
dashboard-1

Output we want:
example-1

### PR DESCRIPTION
**Pull request by wingman47**
**What kind of change does this PR introduce?**

Show letter based avatar for organizations based on their initials without any image similar to what we are shown on the dashboard page.

**Issue Number:**

Fixes #1345

**Snapshots/Videos:**

Before:
image

After:
image

**Summary**

Used dicebear api to fetch profile as it was already used in different parts of the application like dashboard, user profile.

### CODE REVIEW

File: INSTALLATION.md
Comment (SiddheshKukade) [inline INSTALLATION.md:1]: Why these spaces are added here ?

File: src/components/OrgListCard/OrgListCard.tsx [hunk 1]
```diff
@@ -39,21 +39,23 @@ function orgListCard(props: InterfaceOrgListCardProps): JSX.Element {
 
   return (
     <>
+      {console.log('name', name)}
```
Comment (SiddheshKukade) [inline src/components/OrgListCard/OrgListCard.tsx:42]: you can remove the logs.

File: src/components/OrgListCard/OrgListCard.tsx [hunk 2]
```diff
@@ -39,21 +39,23 @@ function orgListCard(props: InterfaceOrgListCardProps): JSX.Element {
...
+                  ? image
+                  : `https://api.dicebear.com/5.x/initials/svg?seed=${name
+                      .split(/\s+/)
+                      .map((word) => word.charAt(0))
+                      .slice(0, 2)
+                      .join('')}`
+              }
+              className={styles.orgimg}
+              alt={`${name} image`}
+            />
```
Comment (SiddheshKukade) [inline src/components/OrgListCard/OrgListCard.tsx:58]: this looks great.

File: src/components/OrgListCard/OrgListCard.tsx (hunk 1 above)
Comment (wingman47) [reply src/components/OrgListCard/OrgListCard.tsx]: Done

File: INSTALLATION.md
Comment (wingman47) [reply INSTALLATION.md:1]: In Installation.md file?

File: src/components/OrgListCard/OrgListCard.tsx (hunk 2 above)
Comment (wingman47) [reply src/components/OrgListCard/OrgListCard.tsx:57]: Thanks!

File: src/components/OrgListCard/OrgListCard.tsx [hunk 3]
```diff
@@ -42,18 +42,20 @@ function orgListCard(props: InterfaceOrgListCardProps): JSX.Element {
...
+              src={
+                image
+                  ? image
+                  : `https://api.dicebear.com/5.x/initials/svg?seed=${name
+                      .split(/\s+/)
+                      .map((word) => word.charAt(0))
+                      .slice(0, 2)
+                      .join('')}`
+              }
+              className={styles.orgimg}
```
Comment (rishav-jha-mech) [inline src/components/OrgListCard/OrgListCard.tsx:55]: Add a border radius of 4px

**Comment by rishav-jha-mech** [comment]
@wingman47 make the required changes, then the PR is good to be merged

File: src/components/OrgListCard/OrgListCard.tsx (hunk 3 above)
Comment (wingman47) [reply src/components/OrgListCard/OrgListCard.tsx]: The border radius is already set to 4px for `orgImgContainer` class. Should I increase it?
I removed `className={styles.orgimg}` due to the absence of corresponding CSS styles.

Comment (rishav-jha-mech) [reply src/components/OrgListCard/OrgListCard.tsx]: The border radius on orgImgContainer class will not work unless this property is applied to it, 
```css
overflow: hidden;
```
better directly apply border-radius to image, remove the unused CSS too, we don't want any bloat in our code.

Comment (wingman47) [reply src/components/OrgListCard/OrgListCard.tsx]: Done.
image

Comment (wingman47) [reply src/components/OrgListCard/OrgListCard.tsx]: Done
image

File: src/components/OrgListCard/OrgListCard.module.css [hunk 4]
```diff
@@ -21,7 +21,7 @@
   border-radius: 4px;
 }
```
Comment (rishav-jha-mech) [inline src/components/OrgListCard/OrgListCard.module.css:23]: .orgCard .innerContainer .orgImgContainer .emptyImg {
  width: 125px;
  height: 120px;
  background-color: var(--bs-gray-200);
}

You removed this class from the UI but did not remove this from the CSS file

**Review by SiddheshKukade — APPROVED** [review]

**Comment by rishav-jha-mech** [comment]
LGTM

**Review by palisadoes — APPROVED** [review]

**Pull request merged**